tests/metagen/typegraphs/sample/ts_upload/client.ts

```
@@ -880,23 +872,28 @@ type ErrorPolyfill = new (msg: string, payload: unknown) => Error;
+    qg: _QueryGraphBase,
+    addr: URL | string,
+    options?: GraphQlTransportOptions,
+  ) {
     return new GraphQLTransport(
       new URL(addr),
       options ?? {},
-      this.typeNameMapGql,
+      qg.typeNameMapGql,
     );
   }
 }
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)